### PR TITLE
Added documentation for ActiveJob queue adapter for a specific job.

### PR DESCRIPTION
**Pull request by mohitnatoo**
- Took inspiration from the documentation for per job queue configuration. As Rails now has ability to configure queue adapters on a per job basis, thought I'd add up the documentation for the same.

### CODE REVIEW

**Comment by vipulnsward** [comment]
I think you can also add about subclass maintaining this. Just an example would do.

Please add [ci skip] to the commit message

File: guides/source/active_job_basics.md [hunk 1]
```diff
@@ -152,6 +152,21 @@ Here is a noncomprehensive list of documentation:
...
+
+```ruby
+class GuestsCleanupJob < ActiveJob::Base
+  self.queue_adapter = :resque
+  #....
+end
+```
+
+# Now your job will use `resque` as it's backend queue adapter overriding what
+# was configured in `config.active_job.queue_adapter`.
```
Comment (vipulnsward) [inline guides/source/active_job_basics.md]: The comments need to be in ruby block

File: guides/source/active_job_basics.md [hunk 2]
```diff
@@ -152,6 +152,21 @@ Here is a noncomprehensive list of documentation:
 - [Sucker Punch](https://github.com/brandonhilkert/sucker_punch#active-job)
 - [Queue Classic](https://github.com/QueueClassic/queue_classic#active-job)
 
+Job Specific Backend
```
Comment (vipulnsward) [inline guides/source/active_job_basics.md]: See how above sub-heading is added. Please change to follow that- "### Job Specific Backends"

Comment (vipulnsward) [inline guides/source/active_job_basics.md]: Actually, can you drop new subheading and move this to "### Setting the Backend" block? This should follow how we can set global adapter.

**Comment by mohitnatoo** [comment]
@vipulnsward made the changes.

**Comment by vipulnsward** [comment]
Thanks @mohitnatoo !

**Pull request merged**